modules/friendlytag.js

```diff
@@ -53,7 +53,16 @@ Twinkle.tag.callback = function friendlytagCallback() {
 	}).done(function(response) {
 		var isReviewed = false;
 		var isOldPage = response.pagetriagelist.result !== 'success' || response.pagetriagelist.pages.length === 0;
-		var isMarkedAsReviewed = response.pagetriagelist.pages[0].patrol_status > 0;
+		var isMarkedAsReviewed;
+
+		// Handle edge case where pagetriagelist API returns "pages_missing_metadata"
+		var valid_response = response.pagetriagelist.pages[0] !== undefined;
+		if (valid_response) {
+			isMarkedAsReviewed = response.pagetriagelist.pages[0].patrol_status > 0;
+		} else {
+			isMarkedAsReviewed = true;
+		}
```

Comment:
Can be written more concisely:
```suggestion
		// Handle edge case where pagetriagelist API returns "pages_missing_metadata"
		var isMarkedAsReviewed = response.pagetriagelist.pages[0] ?
			response.pagetriagelist.pages[0].patrol_status > 0 : 
			true;
```